test/integration/check/test_check.py

```diff
@@ -0,0 +1,41 @@
+import os
+import re
+
+from utils import run_command_and_get_output
+
+
+def test_check_cmd(tt_cmd, tmpdir):
+    # testing with unset env variable
+    if os.environ.get("TT_CLI_INSTANCE") is not None:
```

Comment:
Since "TT_CLI_INSTANCE" is an `internal' env. var. it can be used in unit testing, but not in integration.